airbyte-webapp/src/components/ui/Switch/Switch.tsx

```diff
@@ -44,7 +44,7 @@ export const Switch: React.FC<SwitchProps> = ({
     <label className={labelStyle}>
       <input
         {...props}
-        aria-checked={indeterminate ? "mixed" : checked}
+        aria-checked={indeterminate ? "mixed" : checked || !!value}
```

Comment:
```suggestion
        aria-checked={(indeterminate ? "mixed" : checked) ?? !!value}
```
This feels slightly safer.